src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -214,12 +215,9 @@ public boolean isStringInURL(String givenString) {
   public void verifyURLcontains(final String givenString, int timeOut) {
     changeImplicitWait(250, TimeUnit.MILLISECONDS);
     try {
-      new WebDriverWait(driver, timeOut).until(new ExpectedCondition<Boolean>() {
-        @Override
-        public Boolean apply(WebDriver driver) {
-          return driver.getCurrentUrl().toLowerCase().contains(givenString.toLowerCase());
-        }
-      });
+      new WebDriverWait(driver, timeOut).until(
+          (ExpectedCondition<Boolean>) driver -> driver.getCurrentUrl().toLowerCase()

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "driver" which hides the field declared at line 45. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)